src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,150 @@
+        try {
+            String taskStr = userInput.substring(4).strip();
+            Task t = taskList.get(Integer.parseInt(taskStr) - 1);
+            t.setDone(true);
+            printOutput("Nice! I've marked this task as done:\n\t" + t.toString(), false);
+        } catch (NumberFormatException e) {
+            printOutput("Oops! Please enter a number for the task!", true);
+        } catch (IndexOutOfBoundsException e){
+            printOutput("Oops! We couldn't find that entry in the list!", true);
+        } catch (Exception e){
+            printOutput("Oops! That didn't work, please check your input and try again!", true);
+        }
+    }
+
+    private static void createDeadline(String userInput) {
+        if (!userInput.toLowerCase().contains("/by")) {
+            printOutput("usage: deadline <name of item> /by <dd-MMM-yyyy HH:mm:ss OR next [weekday]>", true);
+            return;
+        }
+        String[] taskArgs = userInput.substring(8).strip().split("/by");
+        try {
+            Deadline d = new Deadline(taskArgs[0].strip(), taskArgs[1].strip());
+            taskList.add(d);
+            printOutput("Added new deadline: " + taskArgs[0] + System.lineSeparator() + "\tDue by: " + d.getDue());
+        } catch (DukeException de) {
+            printOutput("Unable to parse date! Please try again.", true);
+            return;
+        }
+    }
+
+    private static void createTodo(String userInput) {
+        String taskString = userInput.substring(4).strip();
+        ToDo t = new ToDo(taskString);
+        taskList.add(t);
+        printOutput("Added new todo: " + t.toString());
+    }
+
+    private static void createEvent(String userInput) {
+        if (!userInput.toLowerCase().contains("/at")) {
+            printOutput("usage: event <name of item> /at <dd-MMM-yyyy HH:mm:ss OR next [weekday]>", true);
+            return;
+        }
+        String[] taskArgs = userInput.substring(5).strip().split("/at");
+        try {
+            Event e = new Event(taskArgs[0].strip(), taskArgs[1].strip());
+            taskList.add(e);
+            printOutput("Added new event: " + taskArgs[0] + System.lineSeparator() + "\tStart at: " + e.getStart());
+        } catch (DukeException de) {
+            printOutput("Unable to parse date! Please try again.", true);
+            return;
+        }
+    }
+
+    private static void printOutput(String text) {
+        printOutput(text, false);
+    }
+
+    /**
+     * Prints the output with the divider lines and the supplied text
+     * Option to make the text non instant for extra effect
+     * @param text string to be printed
+     * @param instant whether the string is printed instantly
+     */
+    private static void printOutput(String text, boolean instant) {
+        final String UNDERSCORES = "____________________________________________________________";
+        System.out.println(UNDERSCORES);
+        //Split text according to the lines to format
```

Comment:
Take note. Javadoc comments should be `// Split text according to...... to format.`. Be careful of the space after the `//` and a `.` at the end of the sentence.

Reply:
The same also applies to all other instances where you used a `//` comment.